src/framework/audio/engine/internal/tracksequence.cpp

```
@@ -38,13 +38,13 @@ using namespace muse::async;
 using namespace muse::audio;
 using namespace muse::audio::engine;
 
-TrackSequence::TrackSequence(const TrackSequenceId id, const modularity::ContextPtr& iocCtx)
-    : muse::Contextable(iocCtx), m_id(id)
+TrackSequence::TrackSequence(const TrackSequenceId id)
+    : m_id(id)
 {
     ONLY_AUDIO_ENGINE_THREAD;
 
     m_clock = std::make_shared<Clock>();
```

Cppcheck (2.20.0)

[performance] Variable 'm_clock' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)